app/src/main/res/color/color_navigation_view_icon.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
     <item android:color="@color/appcompat_secondary_text_disabled_material_light" android:state_enabled="false"/>
-    <item android:color="?colorPrimary" android:state_checked="true"/>
+    <item android:color="@android:color/black" android:state_checked="true"/>
```

Comment:
Well it doesn't work on 5.1.1 for me and since using the primary color is almost identical black is better.